ONSdigital/dp-elasticsearch
# Signing Elasticsearch Requests using AWS SDK

### What

Provide working solution to sign Http requests to AWS managed elasticsearch service using AWS SDK to retrieve credentials using several known providers and the V4 AWS SDK signer method.

Total removal of smartystreets sign method

awsauth package will sit here until we decide to move it to new library, solution is not Elasticsearch specific.

### How to review

- Check changes make sense
- Unit tests pass
- Check documentation in README.md is clear

### Who can review

Anyone
